### ISSUE TITLE
network shows disconnected instead of connected with NetworkManager

### ISSUE THREAD

**Issue by kevinoid** [Alexays/Waybar#1129]
Testing with `master` (9e34be7) I noticed another issue similar to #1117, except now the disconnected ethernet Interface is shown rather than the connected wireless interface:

On a computer running NetworkManager with both a wireless interface (`wlp3s0`) and an ethernet interface (`enp0s25`), when the ethernet interface is disconnected, Waybar displays information for the disconnected ethernet interface rather than the connected wireless interface.  To reproduce, using the following `config`:

```json
{
  "modules-left": [
    "network"
  ],
  "network": {
    "format": "{ifname}:{ipaddr}/{cidr}",
    "format-disconnected": "{ifname}:Disconnected"
  }
}
```

1. Start with ethernet connected and wireless disconnected.
   `nmcli device` reports:

       DEVICE   TYPE      STATE         CONNECTION 
       enp0s25  ethernet  connected     EthConn  
       wlp3s0   wifi      disconnected  --         
       lo       loopback  unmanaged     --         

   `ip -4 -br addr` reports:

       lo               UNKNOWN        127.0.0.1/8 
       enp0s25          UP             192.168.21.22/24 

   Start Waybar: `waybar -c config -l trace`.
   Waybar shows `enp0s25:192.168.21.22/24`
2. Connect to a wireless network (`nmcli device connect wlp3s0`).
   `nmcli device` reports:

       DEVICE   TYPE      STATE      CONNECTION 
       enp0s25  ethernet  connected  EthConn  
       wlp3s0   wifi      connected  WifiConn 
       lo       loopback  unmanaged  --         

   `ip -4 -br addr` reports:

       lo               UNKNOWN        127.0.0.1/8 
       enp0s25          UP             192.168.21.22/24 
       wlp3s0           UP             192.168.21.10/24 

   Waybar shows `enp0s25:192.168.21.22/24`
3. Disconnect the ethernet cable.
   `nmcli device` reports:

       DEVICE   TYPE      STATE        CONNECTION 
       wlp3s0   wifi      connected    WifiConn 
       enp0s25  ethernet  unavailable  --         
       lo       loopback  unmanaged    --         

   `ip -4 -br addr` reports:

       lo               UNKNOWN        127.0.0.1/8 
       wlp3s0           UP             192.168.21.10/24 

   Waybar shows `wlp3s0:Disconnected`

After step 3, I would have expected `wlp3s0:192.168.21.10/24`, since NetworkManager and `ip` show it as connected.

Capture of `stdout` with `-l trace` during the procedure described above:

    [2021-06-07 10:23:51.803] [info] Resources files: config, /home/kevin/.config/waybar/style.css
    [2021-06-07 10:23:51.811] [debug] Output detection done: LVDS-1 (Unknown 0x213E 0x00000000)
    [2021-06-07 10:23:51.814] [debug] network: new default route via if2 metric 100
    [2021-06-07 10:23:51.814] [debug] network: enp0s25, new addr 192.168.21.22/24
    [2021-06-07 10:23:51.846] [debug] GTK widget tree:
    window#waybar.background.LVDS-1.:dir(ltr)
      decoration:dir(ltr)
      box.horizontal:dir(ltr)
        box.horizontal.modules-left:dir(ltr)
          widget:dir(ltr)
            label#network:dir(ltr)
        box.horizontal.modules-center:dir(ltr)
        box.horizontal.modules-right:dir(ltr)
    
    [2021-06-07 10:23:51.848] [info] Bar configured (width: 1600, height: 17) for output: LVDS-1
    [2021-06-07 10:24:56.816] [debug] network: default route deleted enp0s25/if2 metric 100
    [2021-06-07 10:24:56.818] [debug] network: new default route via if3 metric 600
    [2021-06-07 10:24:56.818] [debug] network: wlp3s0, new addr 192.168.21.10/24

Thanks again for working on it!
Kevin

**Comment by tperard** [comment]
Hi @kevinoid, thanks again!

I hope that this commit fix the issue you are seeing :-).

**Comment by kevinoid** [comment]
Everything looks great with 2016074.  Thanks again for working on it @tperard!
